android/assets/jsons/translations/template.properties

```diff
@@ -597,6 +605,7 @@ Maintenance cost =
 Pick construction = 
 Pick improvement = 
 Provides [resource] = 
+Provides [amount] [resource] = 
```

Comment:
I'm pretty sure this is already translated, no? It should be an atom bomb requirement

Reply:
Oh no that was the 'requires'
nvm